### PR TITLE
Feat/alert

### PR DESCRIPTION
**Pull request by ajay-mandaviya**
Screenshot from 2022-02-05 14-23-37

### CODE REVIEW

**Review by SukanyaSen530 — COMMENTED** [review]
PR description is missing.

**Review by tanishk15sharma — APPROVED** [review]
I liked the naming convention .

**Review by prathmeshjagtap — COMMENTED** [review]
code looks good and clean

File: component/alert/alert.css [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+  width: 100%;
+  display: flex;
+  align-items: center;
+  justify-content: flex-start;
+  padding: 0.5rem 1rem;
+  border-radius: 5px;
+  margin-top: 20px;
+}
+.alert > div {
+  margin-right: 20px;
```
Comment (amarphule) [inline component/alert/alert.css:18]: you can use `rem` instead `px`

**Review by amarphule — COMMENTED** [review]
code looks good to me. One suggestion you can use `rem` instead `px`.

**Pull request merged**